package org.example.examClouds.Lesson19.uncontrolledException;

public class ExceptionDemo2 {
    public static void main(String[] args) {
        int[] array = new int[3];
        System.out.println(array[4]);
    }
}
